src/lib/fetchAPI.js

```diff
@@ -0,0 +1,34 @@
+async function requestLogin(email, password) {
+  const res = await fetch("https://eventmakers-api.fly.dev/auth/login", {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify({
+      email,
+      password,
+    }),
+  });
+
+  const data = await res.json();
+  return data;
+}
+
+async function requestRegister(name, email, password) {
+  const res = await fetch("https://eventmakers-api.fly.dev/auth/register", {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify({
+      name,
+      email,
+      password,
+    }),
+  });
+
+  const data = await res.json();
+  return data;
+}
+
+export { requestLogin, requestRegister };
```

Comment:
Remove these and convert to `export async function`

Reply:
done